trunk/SUAVE/Analyses/Weights/Weights_electric_helicopter.py

```diff
@@ -0,0 +1,55 @@
+## @ ingroup Analyses-Weights
```

Comment:
Have you tested that @ ingroup works with the space?

Reply:
Space removed.